### PR TITLE
update vere reference for 1.15

### PR DESCRIPTION
**Pull request by tinnus-napbus**
this updates the runtime reference with new utilities `eval` and `vile`, as well as the new loom options

### CODE REVIEW

File: content/manual/running/vere.md [hunk 1]
```diff
@@ -197,6 +198,29 @@ command.
...
+Evaluate a hoon expression without booting a ship.
+
+The expression to evaluate is given in a string via stdin, and the result is
+pretty-printed to the terminal. Note you do not need to boot an actual ship to
+run this, the runtime can do it itself.
+
+- Undocked: `echo [expression] | urbit eval`
+- Docked: `echo [expression] | [pier]/.run eval`
+
+This will work like:
```
Comment (sigilante) [inline content/manual/running/vere.md:212]: Is this just against the standard subject then, or the current Dojo subject?

File: content/manual/running/vere.md [hunk 2]
```diff
@@ -290,14 +314,24 @@ previously with `cram` or `-n`.
...
 - `-v, --version VER` - version number, `VER` is e.g. `1.9`.
 - `-p, --pace` - release channel, e.g. `live`.
 
 Example usage:
 
 - Undocked: `urbit vere -a x86_64-linux -v 1.9 -p live .`
 
+### `vile`
+
+Export keyfile.
```
Comment (sigilante) [inline content/manual/running/vere.md:328]: What's the expected utility of `vile`?

**Review by sigilante — CHANGES_REQUESTED** [review]
Otherwise LGTM

File: content/manual/running/vere.md (hunk 1 above)
Comment (tinnus-napbus) [reply content/manual/running/vere.md:212]: no it looks like it's eth & azimuth utilties. Idk where it comes from or its purpose

```
❯ echo "." | urbit eval
loom: mapped 2048MB
lite: arvo formula 11a9e7fe
lite: core 38d4ad4d
lite: final state 38d4ad4d
eval:
< 1.mrs
  [   ethereum
    < 12.zya
      17.ayh
      34.bxr
      14.yze
      54.jwx
      77.kdc
      232.gkb
      51.qbt
      123.ppa
      46.hgz
      1.pnw
      %140
    >
      azimuth
    < 6.svq
      11.gpf
      [ < 4.kqd
          34.bxr
          14.yze
          54.jwx
          77.kdc
          232.gkb
          51.qbt
          123.ppa
          46.hgz
          1.pnw
          %140
        >
        < 3.hqp
          34.bxr
          14.yze
          54.jwx
          77.kdc
          232.gkb
          51.qbt
          123.ppa
          46.hgz
          1.pnw
          %140
        >
          ethereum
        < 12.zya
          17.ayh
          34.bxr
          14.yze
          54.jwx
          77.kdc
          232.gkb
          51.qbt
          123.ppa
          46.hgz
          1.pnw
          %140
        >
        < 17.ayh
          34.bxr
          14.yze
          54.jwx
          77.kdc
          232.gkb
          51.qbt
          123.ppa
          46.hgz
          1.pnw
          %140
        >
      ]
    >
    < 17.ayh
      34.bxr
      14.yze
      54.jwx
      77.kdc
      232.gkb
      51.qbt
      123.ppa
      46.hgz
      1.pnw
      %140
    >
  ]
>
```

File: content/manual/running/vere.md (hunk 2 above)
Comment (tinnus-napbus) [reply content/manual/running/vere.md:328]: idk tbh, I can't think of a non-obscure reason to use it

**Pull request merged**